presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -233,8 +234,21 @@ void PrestoServer::run() {
   registerMemoryArbitrators();
   registerShuffleInterfaceFactories();
   registerCustomOperators();
-  protocol::registerHiveConnectors();
-  protocol::registerTpchConnector();
+
+  protocol::registerConnectorProtocol(
```

Comment:
Do we have to ensure that there is a PrestoToVeloxConnector for each ConnectorProtocol type ? If yes, then maybe we should combine them in a single function.